constellations/gemini-admin/src/utils/inputText.sass

```diff
@@ -14,7 +14,7 @@
         display: flex
```

Comment:
I think the h2 above gets its margins overriden in some pages.
Could you give the h2 a `margin: spacing2x 0`?